{"uid":"aea62be72f5c770e","name":"Log out from Trello account","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_log_out","historyId":"9eddea92aa33bc4ffb54b4788932f508","time":{"start":1736792679574,"stop":1736792712180,"duration":32606},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7fd67cbaac30>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90dc15d58c6c701563d9d064c49ee438\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"90dc15d58c6c701563d9d064c49ee438\")>\nmethod = <function url_matches.<locals>._predicate at 0x7fd67c95fba0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7fd67cbaac30>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90dc15d58c6c701563d9d064c49ee438\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC010\n    @allure.title('Log out from Trello account')\n    def test_log_out(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:291: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1736792679314,"stop":1736792679573,"duration":259},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1736792679574,"stop":1736792679574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7fd67cbaac30>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90dc15d58c6c701563d9d064c49ee438\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"90dc15d58c6c701563d9d064c49ee438\")>\nmethod = <function url_matches.<locals>._predicate at 0x7fd67c95fba0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7fd67cbaac30>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90dc15d58c6c701563d9d064c49ee438\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC010\n    @allure.title('Log out from Trello account')\n    def test_log_out(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:291: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","steps":[{"name":"Log in to Trello account","time":{"start":1736792684574,"stop":1736792712179,"duration":27605},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 291, in test_log_out\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1736792684574,"stop":1736792687991,"duration":3417},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1736792687991,"stop":1736792712178,"duration":24187},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"2b4f9bd27e5109ff","name":"log","source":"2b4f9bd27e5109ff.txt","type":"text/plain","size":63297}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1736792712319,"stop":1736792712391,"duration":72},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC010"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1672-836"},{"name":"thread","value":"3624-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":68,"unknown":0,"total":69},"items":[{"uid":"5de96d63491cc34d","status":"passed","time":{"start":1736730738885,"stop":1736730765194,"duration":26309}},{"uid":"1d9bb134ea6787b6","status":"passed","time":{"start":1736706100094,"stop":1736706126063,"duration":25969}},{"uid":"fdf3dbca801e806b","status":"passed","time":{"start":1736644511728,"stop":1736644537906,"duration":26178}},{"uid":"69d7f7445d87dfd9","status":"passed","time":{"start":1736619702946,"stop":1736619729126,"duration":26180}},{"uid":"8b938eb34dab7806","status":"passed","time":{"start":1736557699674,"stop":1736557725511,"duration":25837}},{"uid":"dfa3c4f52c29bea8","status":"passed","time":{"start":1736533390480,"stop":1736533416551,"duration":26071}},{"uid":"4c25bba68cea78c9","status":"passed","time":{"start":1736471387086,"stop":1736471413210,"duration":26124}},{"uid":"2b5101608ca2a402","status":"passed","time":{"start":1736447121992,"stop":1736447147938,"duration":25946}},{"uid":"d9a162dc2ae3b7ca","status":"passed","time":{"start":1736384863606,"stop":1736384889822,"duration":26216}},{"uid":"c7821ae5401e09aa","status":"passed","time":{"start":1736360595284,"stop":1736360621228,"duration":25944}},{"uid":"dbf74f7eac186ff","status":"passed","time":{"start":1736298485239,"stop":1736298510824,"duration":25585}},{"uid":"2b45f3fc5ab62a75","status":"passed","time":{"start":1736274212407,"stop":1736274238388,"duration":25981}},{"uid":"f86c0538c3a98420","status":"passed","time":{"start":1736212099334,"stop":1736212125366,"duration":26032}},{"uid":"3080f01c5ac26834","status":"passed","time":{"start":1736187801611,"stop":1736187827886,"duration":26275}},{"uid":"a9cfa3f7759597c7","status":"passed","time":{"start":1736125845167,"stop":1736125871305,"duration":26138}},{"uid":"519f810416d3d7fd","status":"passed","time":{"start":1736101269356,"stop":1736101295332,"duration":25976}},{"uid":"e468066d70feeb80","status":"passed","time":{"start":1736039571315,"stop":1736039596734,"duration":25419}},{"uid":"7f5e9cc245943688","status":"passed","time":{"start":1736014896403,"stop":1736014922409,"duration":26006}},{"uid":"450aff1ca2e0a468","status":"passed","time":{"start":1735952801364,"stop":1735952827441,"duration":26077}},{"uid":"aa074a7137663a20","status":"passed","time":{"start":1735928648397,"stop":1735928674968,"duration":26571}}]},"tags":["TC010","TC000"]},"source":"aea62be72f5c770e.json","parameterValues":[]}